### PR TITLE
Pushed all work done till now

### PR DESCRIPTION
**Pull request by fredysomy**

### CODE REVIEW

File: abc.json
Comment (habelMarottipuzha) [inline abc.json:1]: remove file if not required

File: android/app/src/debug/AndroidManifest.xml
Comment (habelMarottipuzha) [inline android/app/src/debug/AndroidManifest.xml:1]: Do we need this? seems like u added internet permission in debug file

File: android/app/src/main/kotlin/com/example/mera_pariwar/MainActivity.kt
Comment (habelMarottipuzha) [inline android/app/src/main/kotlin/com/example/mera_pariwar/MainActivity.kt:1]: rename package and folders to support : **com.socio.parivar** 
app name : **My Parivar**

File: android/app/src/profile/AndroidManifest.xml
Comment (habelMarottipuzha) [inline android/app/src/profile/AndroidManifest.xml:1]: is this file required?

File: custom_package/multi-image-viewer/lib/multi_image_viewer.dart
Comment (habelMarottipuzha) [inline custom_package/multi-image-viewer/lib/multi_image_viewer.dart:1]: instead of using custom_package in folders/packages.. use a meaningful name

File: lib/components/home_components/bottom_nav_bar.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  final VoidCallback onClick;
+
+  BottomNavItem({
+    required this.iconpath,
+    required this.text,
+    required this.onClick,
+  });
+}
+
+class CustomBottomNavigationBar extends StatelessWidget {
```
Comment (habelMarottipuzha) [inline lib/components/home_components/bottom_nav_bar.dart:16]: Use better naming

**Review by habelMarottipuzha — APPROVED** [review]
I am approving and merging changes.. but address those comments. and always raise PR to feature/initial branch.. not to main branch

File: lib/components/home_components/bottom_nav_bar.dart (hunk 1 above)
Comment (fredysomy) [reply lib/components/home_components/bottom_nav_bar.dart:16]: Done

File: custom_package/multi-image-viewer/lib/multi_image_viewer.dart
Comment (fredysomy) [reply custom_package/multi-image-viewer/lib/multi_image_viewer.dart:1]: Done

File: android/app/src/profile/AndroidManifest.xml
Comment (fredysomy) [reply android/app/src/profile/AndroidManifest.xml:1]: Its done by flutter ig

File: android/app/src/main/kotlin/com/example/mera_pariwar/MainActivity.kt
Comment (fredysomy) [reply android/app/src/main/kotlin/com/example/mera_pariwar/MainActivity.kt:1]: Done

File: android/app/src/debug/AndroidManifest.xml
Comment (fredysomy) [reply android/app/src/debug/AndroidManifest.xml:1]: To provide hot reload, done by flutter

File: abc.json
Comment (fredysomy) [reply abc.json:1]: Done
